### PR TITLE
[UNR-4658] Remove functional test CI maps

### PR DESCRIPTION
**Pull request by mironec**
https://github.com/spatialos/UnrealGDK/pull/2859

### CODE REVIEW

**Comment by mironec** [comment]
@KeyMaster- your tests are here

File: Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.Build.cs [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
 			"GameplayAbilities",
 			"GameplayTags",
 			"GameplayTasks",
+			"SpatialGDK",
 			"SpatialGDKFunctionalTests"
-        });
-    }
+		});
+	}
 }
```
Comment (KeyMaster-) [inline Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.Build.cs:21]: Github is complaining about the lack of a newline (idk if that's new)

File: Game/Source/GDKTestGymsFunctionalTests/TestMaps/Spatial2WorkerTestGymMap.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+USpatial2WorkerTestGymMap::USpatial2WorkerTestGymMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE_SPATIAL_ONLY, TEXT("Spatial2WorkerTestGymMap"))
+{
+}
+
+void USpatial2WorkerTestGymMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+
+	//AddActorToLevel<APredictedGameplayCuesTest>(CurrentLevel, FTransform::Identity); //Disabled for now - TODO: UNR-4833
```
Comment (KeyMaster-) [inline Game/Source/GDKTestGymsFunctionalTests/TestMaps/Spatial2WorkerTestGymMap.cpp:18]: Best way to disable a test is to never create it. Love it

**Review by KeyMaster- — APPROVED** [review]
One minor comment

**Pull request merged**